### DISCUSSION TITLE
Request Blocked - We believe this transaction is malicious and unsafe to sign. Approving may lead to loss of funds

### DISCUSSION THREAD

**Discussion by cryptonazgul** [phantom/docs#331] [Help & Support] [answered]
### Summary

I am trying to buy some tokens from dex, using base network. I do have enough eth but the transaction is not going through. I tried the same on Uniswap as well and in phantom wallet the request gets blocked

### Steps to Reproduce

N/A

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@cryptonazgul For this request please contact our support team directly via settings in your Phantom wallet.

**Reply by ok872075** [reply]
they added me into their blocklist , is there any information you can help me with?

**Comment by Vlada3333** [comment]
Thank you, I received an answer.

**Reply by ok872075** [reply]
So, did they solved your problem and if they did how long did they take to solve?
